documentation/commands/README.md

```diff
@@ -1,3 +1,475 @@
+
+The option `-i` makes a guess what type you are looking for.
+
+```$ cg get -i TEXT```
+
+where `TEXT` is a familyID, flowcellID, or sampleID.
+
+### cg get family
+Get information about a family.
+
+```$ cg get family FAMILY_ID```
+
+where `FAMILY_ID` is the familyID, e.g. `cuddlygull`.
+
+The same information can be reached using options `--customer` and `--name`, instead of the familyID. Example:
+
+```$ cg get family --name FAMILY_NAME --customer CUSTOMER_ID```
+
+where `FAMILY_NAME` is the customer's internal familyID.
+
+It is possible to toggle if related samples should be shown or not, using `--samples` or `--no-samples`. Example:
+
+```$ cg get family --no-samples FAMILY_ID```
+
+By default the related samples are shown.
+
+### cg get flowcell
+Get information about a flowcell and the samples on it.
+
+```$ cg get flowcell FLOWCELL_ID```
+
+It is possible to toggle if related samples should be shown or not , using `--samples` or `--no-samples`. Example:
+
+```$ cg get flowcell --no-samples FLOWCELL_ID```
+
+By default the related samples are shown.
+
+### cg get sample
+Get information about a sample.
+
+```$ cg get sample SAMPLE_ID```
+
+It is possible to toggle if related families should be shown or not , using `--families` or `--no-families`. Example:
+
+```$ cg get sample --no-families SAMPLE_ID```
+
+By default the related families are shown.
+
+It is also possible to display related flowcells with the `--flowcells` option.
+
+```$ cg get sample --flowcells SAMPLE_ID```
+
+## cg init
+
+*Do* **not** *touch this, please.*
+
+Setup the database.
+
+```$ cg init```
+
+Options for resetting database before setting up tables (`--reset`) and for bypassing manual confirmation (`--force`).
+
+## cg set
+Update information in statusDB.
+
+### cg set family
+Update information about a family.
+
+```$ cg set family FAMILY_ID```
+
+where `FAMILY_ID` is the case name, e.g. `cuddlygull`.
+
+To update the analysis action for the family, use
+
+```$ cg set family --action ACTION FAMILY_ID```
+
+where `ACTION` can be `analyze`, `running`, or `hold`.
+
+To update the priority of the analysis, use
+
+```$ cg set family --priority PRIORITY FAMILY_ID```
+
+where `PRIORITY` can be `research`, `standard`, `priority`, or `express`.
+
+To update the gene panel for an analysis, use
+
+```$ cg set family --panel PANEL FAMILY_ID```
+
+where `PANEL` are the gene panels.
+
+### cg set sample
+Update information about a sample.
+
+```$ cg set sample SAMPLE_ID```
+
+To update the gender of a sample, use
+
+```$ cg set sample --sex SEX```
+
+where `SEX`can be `male`, `female`, or `unknown`.
+
+## cg status
+View status of things.
+
+### cg status analysis
+Shows which families will be analyzed.
+
+```$ cg status analysis```
+
+### cg status families
+View status of families.
+
+```$ cg status families```
+
+There is an option to skip initial records:
+
+```$ cg status families --skip INTEGER```
+
+where `INTEGER` is **yeah, what is it?**.
```

Comment:
No idea man

Reply:
aha, by default it will show 30 families. Skip will skip the first `skip` families.